windows/src/desktop/kmshell/main/OnlineUpdateCheck.pas

```diff
@@ -706,4 +975,426 @@ function TOnlineUpdateSharedData.Params: TOnlineUpdateCheckParams;
   Result := FParams;
 end;
 
+
+function SetBackgroundState(Update : TUpdateState): Boolean;  // I2329
+var
+  UpdateStr : string;
+begin
+
+  Result := False;
+  // TODO Ask Marc how to handle the execption and return false
```

Comment:
This looks like you are asking me something, but I'm not sure what you info you are looking for